app/views/personal_access_tokens/_personal_access_tokens_tab.html.erb

```diff
@@ -0,0 +1,61 @@
+<div class="tab-pane" id="personal_access_tokens">
+  <% if flash[:personal_access_token] %>
```

Comment:
why not use a simple class / attribute to identify this? I'm uncertain on the usage of flash?